server/controllers/commentController.js

```diff
@@ -33,6 +34,7 @@ export default class CommentController {
       })
       .then((comment) => {
         if (comment) {
+          Notification.notifyLikers(data.articleSlug, data.userId);
```

Comment:
When a new comment is added to an article, notify all users that liked the article.